backend/src/test/java/org/josh/backend/workshop/WorkshopControllerTest.java

```diff
@@ -152,4 +152,45 @@ void expectUpdatedWorkshop_whenUpdateWorkshop() throws Exception {
             .andExpect(MockMvcResultMatchers.status().isOk())
             .andExpect(MockMvcResultMatchers.content().json(expect));
     }
+
+    @DirtiesContext
+    @Test
+    void expectEmptyList_whenDeleteWorkshopById() throws Exception {
+
+        //GIVEN
+        String result = mockMvc.perform(
+                MockMvcRequestBuilders.post("/api/workshop")
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(testWorkshopWithoutIdAndLikes))
+            .andReturn().getResponse().getContentAsString();
+
+        Workshop saveResultWorkshop = objectMapper.readValue(result, Workshop.class);
+        String id = saveResultWorkshop.id();
+
+        //WHEN
+        mockMvc.perform(MockMvcRequestBuilders.delete("/api/workshop/%s".formatted(id)))
```

Comment:
What does .formatted() do?

Reply:
replaces the %s with a String. Use %i for integer and so on :v: